roles/alexandria-v2/templates/environment.rb.j2

```diff
@@ -0,0 +1,7 @@
+# Load the Rails application.
+require File.expand_path('../application', __FILE__)
+
+Rails.env = ENV['RAILS_ENV'] || '{{ rails_env }}'
```

Comment:
Yeah, thanks. A separate PR with a link to one in alexandria-v2 would make the change clearer and easier to discuss.